Changelog.rst

```diff
@@ -2,6 +2,21 @@
 Changelog
 =========
 
+1.5.0
+-----
+
+* Dropped support for old Python/Django versions.
+* Added support for ``NullBooleanField`` -- thanks to @ashwch
```

Comment:
Need note for the new guarantee that the values attribute preserves definition order.